src/gengowatcher/config.py

```
@@ -1,5 +1,8 @@
 import configparser
+if sys.platform != 'win32':
```

Ruff (0.14.14)

[error] Undefined name `sys`

(F821)